### PR TITLE
Rb 4.1.x fbcp

### PR DESCRIPTION
**Pull request by ian57**
Please make sure your PR is ready to be merged !
- [x] You added the changes in CHANGELOG.md
- [x] You choose the right repository branch to make the PR
- [x] You described the PR as below

Fixes #XXXX

Changes :
fbcp : Raspberry Pi Framebuffer Copyhttps://github.com/tasanakorn/rpi-fbcp . This program used for copy primary framebuffer to secondary framebuffer (eg. FBTFT). It is very useful for displaying recalbox on small TFT screen like Adafruit 2.8 or 3.2" or Chinese clones. 
- Adding packages/rpi-fbcp directory + Config.in rpi-fbcp.mk
- Modification of recalbox-rpi1_defconfig recalboxrecalbox-rpi3_defconfig-rpi3_defconfig
- Modification of packages/Config.in to display a new item in make menuconfig
- Related to (put here the others PR in other repositories)

Related to (put here the others PR in other repositories)

### CODE REVIEW

File: package/rpi-fbcp/rpi-fbcp.mk [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#
+################################################################################
+
+RPI_FBCP_VERSION = 8087a71d0330a078d91aa78656684ab5313616c6
+RPI_FBCP_SITE = $(call github,tasanakorn,rpi-fbcp,$(RPI_FBCP_VERSION))
+
+define RPI_FBCP_CROSS_FIXUP
+        $(SED) 's|/opt/vc/include|$(STAGING_DIR)/usr/include|g' $(@D)/CMakeLists.txt 
+        $(SED) 's|/opt/vc/lib|$(STAGING_DIR)/usr/lib|g' $(@D)/CMakeLists.txt 
+        $(SED) 's|bcm_host|bcm_host vchostif|g' $(@D)/CMakeLists.txt
```
Comment (digitalLumberjack) [inline package/rpi-fbcp/rpi-fbcp.mk:14]: If it uses bcmhost, you should add rpi-userland in the package dependencies :)

Comment (ian57) [inline package/rpi-fbcp/rpi-fbcp.mk:14]: Thanks for the advice ;)

**Comment by digitalLumberjack** [comment]
@ian57 what is the state of this pr ? Is it mergable ?

**Comment by ian57** [comment]
Not yet, we have some work to integrate it into recalbox as a new feature which can be activable in recalbox.conf. I need to add my /etc/init.d/S11fbcp and some overlays for chinese tft screen.

But if you want bbcp package can be merge alone.

**Comment by digitalLumberjack** [comment]
I close @ian57  you will have to reopen one PR :)

**Comment by ian57** [comment]
Yep no problem.... this PR is not complete and takes too long :)

**Pull request closed without merging**